src/geophires_x/Units.py

```diff
@@ -313,3 +315,13 @@ class HeatPerUnitAreaUnit(str,Enum):
 class PowerPerUnitAreaUnit(str,Enum):
     """Population Density Units"""
     MWPERSQKM = "MW/km**2"
+
+
+class HeatPerUnitVolumeUnit(str,Enum):
+    """Population Density Units"""
+    KJPERCUBICKM = "KJ/km**3"
```

Comment:
I believe the SI standard would be `kJ` (lowercase k; https://en.wikipedia.org/wiki/Joule#Multiples) - should we not follow that convention here?